src/server.py

```diff
@@ -436,8 +437,15 @@ def run_seta_details_query():
         parts = d[0].split("'")
         jobtype.append([parts[1], parts[3], parts[5]])
 
+    alljobs = JOBSDATA.jobtype_query()
+
+    # we return low value job list as default, otherwise we return high value jobs
+    if option == 'low':
+        low_value_jobs = [low_value_job for low_value_job in alljobs if
+                          low_value_job not in jobtype]
+        jobtype = low_value_jobs
```

Comment:
We set the 'option' value as default in https://github.com/mozilla/ouija/pull/113/files/100f4fcfd6bf1544287a5261e162ff6b11bbab42#diff-94ab7c76892f0032ee9ad040e3e8f207R419. And because we store high value jobs as default in seta table, so the **if option == 'low':** make more logical sense to me :)